src/sonarwhal-theme/layout/partials/scan-result-item.hbs

```diff
@@ -13,17 +13,14 @@
     <div class="rule-result--details__body">
         <p class="{{../status}}-badge uppercase-text">{{../status}}</p>
         <p>
-            {{message}}
+            {{{linkify message}}}
         </p>
         {{#if resource}}
-        <div class="rule-result__code">
-
-            <p>
-                <a target="_blank" rel="noopener noreferrer" href="{{resource}}">
-                    {{cutUrlString resource}}{{normalizePosition location.line}}{{normalizePosition location.column}}
-                </a>
-            </p>
-        </div>
+        <p>
```

Comment:
Why are you removing the `<div>` from here?

Reply:
@molant Because currently in the result page the url is also wrapped in a `code` block.

![resource_before](https://user-images.githubusercontent.com/20218531/35818086-f6cd4e64-0a53-11e8-9940-d5b18b4b1791.PNG)

After removing the <div> it is back to 

![resource_after](https://user-images.githubusercontent.com/20218531/35818113-0655c79e-0a54-11e8-92e1-a2b47e3ea353.PNG)